test/create.test.js

```diff
@@ -43,7 +46,13 @@ describe('create', function() {
     }
     Foo.prototype.b = 2;
 
-    assert.deepStrictEqual(create({}, new Foo), { 'a': 1, 'c': 3 });
+    var actual = create({}, new Foo);
+    var expected = { 'a': 1, 'c': 3 };
+
+    assert.deepStrictEqual(Object.keys(actual), Object.keys(expected));
+    Object.keys(expected).forEach(property => {
```

Comment:
I see in a couple places we do `Object.keys(expected)` would it make sense to store the result in a var when possible?

Reply:
done